opentreemap/treemap/js/test/urlState.js

```diff
@@ -130,15 +130,15 @@ var urlStateTests = {
         var context = createContext();
         urlState.init(context);
         urlState.stateChangeStream.take(1).onValue(function(state) {
-            assert.deepEqual(state, {bar: 2});
+            assert.deepEqual(state, {bar: '2'});
```

Comment:
Adding in the line `newState = getStateFromUrl(getUrlFromState(newState));` means that we always serialize our URL state to text and then deserialize it back to a JS object.  For special-cased items that have a deserialize function builtin, like `zoomLatLng` we can convert it to a number, but for unrecognized items in the URL they'll always be text.

This was the case before when we loaded the page with those items in the URL, so this isn't changing behaviour too drastically.